legal-api/src/legal_api/reports/report.py

```diff
@@ -679,6 +679,31 @@ def _format_alteration_data(self, filing):
             if new_legal_type else None
 
     def _format_amalgamation_data(self, filing):
+
+        filings = Filing.get_filings_by_types(self._business.id, ['amalgamationApplication'])
```

Comment:
you don't need get the filing again. use self._filing (https://github.com/bcgov/lear/blob/main/legal-api/src/legal_api/reports/report.py#L46). FYI in PAID state _business will be none, so this will lead to exception